chrome/content/ui.js

```diff
@@ -377,6 +377,9 @@ var ui = {
       callback: function() {
         let name = uiConv.target.normalizedName;
         ui.openAuth(window, otrAuth, name, "start", uiConv);
+
+		// prevent closing of notification bar when the button is hit
+		throw new Error('prevent nb close');
         return false;
```

Comment:
Throwing an error is undesirable, but thankfully I think returning `true` here seems to work.